lib/screens/app/guardians/my_guardians_tab.dart

```diff
@@ -20,52 +20,137 @@ class MyGuardiansTab extends StatelessWidget {
-                  child: const Text('Remove Guardian', style: TextStyle(color: Colors.red)),
-                  onPressed: () {
-                    FirebaseDatabaseService().removeMyGuardian(
-                        currentUserId: SettingsNotifier.of(context).accountName, friendId: user.account);
-                    Navigator.pop(context);
-                  },
-                )
-              ],
-            ));
+    _onTileTapped(MemberModel user, Guardian guardian) {
+      if (guardian.recoveryStartedDate != null) {
+        showRecoveryStartedBottomSheet(context, user);
+      } else {
+        if (guardian.status == GuardianStatus.alreadyGuardian) {
+          showGuardianOptionsDialog(context, user);
+        }
       }
     }
 
-    List<Widget> items = [];
-    if (myGuardians.where((element) => element.status == GuardianStatus.alreadyGuardian).length < 3) {
-      items.add(Container(
-        color: Colors.white,
-        child: Padding(
-          padding: const EdgeInsets.all(16.0),
-          child: Center(
-            child: Text(
-              "IMPORTANT: You need a minimum of 3 Guardians to secure your backup key",
-              style: TextStyle(color: Colors.red, fontSize: 16),
+    if (myMembers.isEmpty) {
+      return Center(
+          child: Padding(
+        padding: const EdgeInsets.all(32.0),
+        child: Text("You have added no user to become your guardian yet. Once you do, the request will show here."),
+      ));
+    } else {
+      List<Widget> items = [];
+      if (myGuardians.where((element) => element.status == GuardianStatus.alreadyGuardian).length < 3) {
+        items.add(Container(
+          color: Colors.white,
+          child: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Center(
+              child: Text(
+                "IMPORTANT: You need a minimum of 3 Guardians to secure your backup key",
+                style: TextStyle(color: Colors.red, fontSize: 16),
+              ),
             ),
           ),
-        ),
-      ));
+        ));
+      }
+      items.add(Expanded(
+          child: buildMyGuardiansListView(
+              myMembers, SettingsNotifier.of(context).accountName, myGuardians, _onTileTapped)));
+
+      return Column(children: items);
     }
+  }
 
-    items.add(Expanded(
-        child:
-            buildMyGuardiansListView(myMembers, SettingsNotifier.of(context).accountName, myGuardians, _onTileTapped)));
+  void showGuardianOptionsDialog(BuildContext context, MemberModel user) {
+     showDialog(
+        context: context,
+        child: AlertDialog(
+          content: Text("Guardian ${user.nickname}"),
+          actions: [
+            FlatButton(
+              child: const Text('Dismiss'),
+              onPressed: () {
+                Navigator.pop(context);
+              },
+            ),
+            FlatButton(
+              child: const Text('Remove Guardian', style: TextStyle(color: Colors.red)),
```

Comment:
I guess all of them should be .i18n

Reply:
Yes yes, I am gonna have my brother do the translation work :)